### PR TITLE
Custom sku fix

### PR DESCRIPTION
**Pull request by sebbean**
fixed bug with rows / headers not reading properly
import no longer imports duplicate products when custom sku exist

### CODE REVIEW

File: lib/spree_importer/importers/variant.rb [hunk 1]
```diff
@@ -42,7 +44,7 @@ def import(headers, csv)
       end
 
       def master_variant(headers, row)
-        target.find_by_sku val(headers, row, "master_sku")
+        target.find_by_sku val(headers, row, "master-sku")
```
Comment (weeksie) [inline lib/spree_importer/importers/variant.rb]: Will the hyphen in `master-sku` cause the same issues with delimiters that the underscore did?

File: spec/lib/spree_importer/importers/variant_spec.rb [hunk 2]
```diff
@@ -14,7 +16,7 @@
   it "should import stock items in the proper quantity" do
     import_source_file = get_import_source_file "gin-lane-variant-export"
     import_source_file.import!
-    variant         = Spree::Variant.find_by_sku "STN-FW13-DUMMY-NO-SIZE"
+    variant         = Spree::Variant.find_by_sku "STN-FW13-DUMMY-NO-COLOR"
```
Comment (weeksie) [inline spec/lib/spree_importer/importers/variant_spec.rb:19]: Ha! Thanks for fixing this. So fucking lazy of me.

File: lib/spree_importer/importers/variant.rb (hunk 1 above)
Comment (sebbean) [inline lib/spree_importer/importers/variant.rb]: so 'master-sku' gets converted to 'master sku' which is the real column name in the csv (updated those as well)

Comment (sebbean) [inline lib/spree_importer/importers/variant.rb]: k scottyo got your symbols in there

**Pull request merged**